### ISSUE TITLE
An error occures during the 'precise32' download

### ISSUE THREAD

**Issue by attillax** [phretor/memory-errors-lab#6]
```
attillax@Asus:~/Lernen/PaS/lab1/phretor/memory-errors-lab/linux64$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Box 'precise32' could not be found. Attempting to find and install...
    default: Box Provider: virtualbox
    default: Box Version: >= 0
==> default: Box file was not detected as metadata. Adding it directly...
==> default: Adding box 'precise32' (v0) for provider: virtualbox
    default: Downloading: precise32
An error occurred while downloading the remote file. The error
message, if any, is reproduced below. Please fix this error and try
again.

Couldn't open file /home/attillax/Lernen/PaS/lab1/phretor/memory-errors-lab/linux64/precise32
```

**Comment by phretor** [comment]
Can you try now with the latest commit and let me know if that works?

**Comment by attillax** [comment]
```
attillax@Asus:~/Lernen/PaS/lab1/phretor/memory-errors-lab/linux64$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Checking if box 'debian/jessie64' is up to date...
==> default: Fixed port collision for 22 => 2222. Now on port 2201.
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 22 => 2201 (adapter 1)
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2201
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Connection timeout. Retrying...
    default: 
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default: 
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
```

It seems that everything is works now

**Comment by phretor** [comment]
Fixed by 03b06b8aaad84d2801934b3b87fd166f9e99069c
